designs/virtualization/4.5/Scheduling/CPU-features/cpu-features.md

```diff
@@ -0,0 +1,20 @@
+---
+parent: virtualization
+version: 4.5
+---
+
+# Fine Grained CPU Features
+
+Enabling fine grained features for VM.
+
+![VM page CPU features default](img/CPU-features-0-0.jpg)
+CPU Features is placed in the Scheduling section of the VM Details tab.
+
+![CPU features modal](img/CPU-features-1-0.jpg)
+Currently, these features will be available to be included ot excluded.
+Any advanced options or differet stautes (Optional, Focred...) are out of scope.
```

Comment:
Should we say they are out of scope here? Maybe we can drop this line entirely.

Reply:
I was going to say for the documentation we just drop this for now. We can always extend the documentation with a new PR when/if we decide to add it as a feature.